src/cascadia/TerminalApp/TerminalPage.idl

```diff
@@ -1,10 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import "..\App.idl";
```

Comment:
Why is this needed?  Is this for the definition of LastTabClosedEventArgs?  Can that be moved into this file?

Reply:
Moved

Reply:
You said Moved, but I cannot figure out whether it's actually been moved. Page is still including App.idl?